src/ReactiveUI/Activation.cs

```diff
@@ -54,6 +54,9 @@ public sealed class ViewModelActivator
         /// <value>The deactivated.</value>
         public IObservable<Unit> Deactivated { get { return deactivated; } }
 
+        /// <summary>
```

Comment:
Comments like this always feel...pointless, happy to remove it if wanted but thought since I'm here I may as well fix the compile time warning...